packages/core/documentation/src/index.ts

```diff
@@ -0,0 +1,25 @@
+// import type { App } from 'vue'
+import DocumentationContent from './components/DocumentationContent.vue'
+import DocumentationDisplay from './components/DocumentationDisplay.vue'
+import DocumentationPageEmptyState from './components/DocumentationPageEmptyState.vue'
+import ProductDocumentModal from './components/ProductDocumentModal.vue'
+
+// Export Vue plugin
+// We rarely want to export components as a plugin as we prefer to support proper tree-shaking in the host application. Only enable if you're packing a Vue plugin.
+// export default {
+//   // Customize Vue plugin options as desired
+//   // Providing a `name` property allows for customizing the registered
+//   // name of your component (useful if exporting a single component).
+//   install: (app: App, options: { name?: string, [key: string]: any } = {}): void => {
+//     app.component(options.name || 'Documentation', Documentation)
+//   },
+// }
```

Comment:
nit: you can remove the plugin export if not needed

Reply:
fixed